src/frontend-pwa/src/components/SettingsNavButton/SettingsNavButton.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * @summary Reusable settings button navigation component
+ * @author Dallas Richmond
+ */
+import { useLocation } from 'react-router-dom';
+import gear from '/gear-icon.svg';
+import {
+  StyledSettingsButton,
+  StyledIcon,
+} from './settingsNavButton.styles';
+
+export default function BackNavButton() {
```

Comment:
Is this correctly named or should it be SettingsNavButton?

Reply:
@BradyMitch Thanks for the catch!